modules/apps/frontend-theme/frontend-theme-dialect/src/WEB-INF/frontend-token-definition.json

```diff
@@ -4906,6 +4906,829 @@
+								{
+									"type": "cssVariable",
+									"value": "table-sm-font-size"
+								}
+							],
+							"name": "tableSmFontSize",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--font-weight-normal)",
+							"label": "table-sm-font-weight",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-font-weight"
+								}
+							],
+							"name": "tableSmFontWeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "1.25rem",
+							"label": "table-sm-header-cell-min-height",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-cell-min-height"
+								}
+							],
+							"name": "tableSmHeaderCellMinHeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--spacer-2)",
+							"label": "table-sm-header-cell-padding-x",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-cell-padding-x"
+								}
+							],
+							"name": "tableSmHeaderCellPaddingX",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--spacer-2)",
+							"label": "table-sm-header-cell-padding-y",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-cell-padding-y"
+								}
+							],
+							"name": "tableSmHeaderCellPaddingY",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--text-paragraph-font-size)",
+							"label": "table-sm-header-font-size",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-font-size"
+								}
+							],
+							"name": "tableSmHeaderFontSize",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--font-weight-semi-bold)",
+							"label": "table-sm-header-font-weight",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-font-weight"
+								}
+							],
+							"name": "tableSmHeaderFontWeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--text-paragraph-line-height)",
+							"label": "table-sm-header-line-height",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-header-line-height"
+								}
+							],
+							"name": "tableSmHeaderLineHeight",
+							"type": "String"
+						},
+						{
+							"defaultValue": "var(--text-paragraph-line-height)",
+							"label": "table-sm-line-height",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "table-sm-line-height"
+								}
+							],
+							"name": "tableSmLineHeight",
+							"type": "String"
+						}
+					],
+					"label": "table-sm",
+					"name": "table-sm"
+				},
+				{
+					"frontendTokens": [
+						{
+							"defaultValue": "var(--color-neutral-2)",
+							"editorType": "ColorPicker",
+							"label": "table-spaced-background-color",
```

Comment:
This should probably be table-spaced-row-background-color

Reply:
fixed